src/main/java/trello/entities/List.java

```diff
@@ -110,7 +110,7 @@ public boolean areListsEquals(final ArrayList<Card> listCards) {
      *
      * @return an ArrayList with all cards' titles.
      */
-    public ArrayList<String> getCardsTitles() {
+    public java.util.List<String> getCardsTitles() {
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.